scripts/wireguard/disableCONF.sh

```diff
@@ -113,9 +112,17 @@ done
 
 # Restart WireGuard only if some clients were actually deleted
 if [ "${CHANGED_COUNT}" -gt 0 ]; then
-    if systemctl reload wg-quick@wg0; then
-        echo "::: WireGuard reloaded"
+    if grep -qsEe "^NAME\=['\"]?Alpine[a-zA-Z ]*['\"]?$" /etc/os-release; then
```

Comment:
In this case and similar you can just use `${PLAT}` from setupVars.conf (already sourced).